code/modules/client/preferences_toggles.dm

```diff
@@ -207,10 +208,20 @@
 	set category = "Preferences"
 	set desc = "Toggles movement of parallax space."
 
-	prefs.parallax_togs ^= PARALLAX_IS_STATIC
+	prefs.toggles_secondary ^= PARALLAX_IS_STATIC
 	prefs.save_preferences()
 
-	if (prefs.parallax_togs & PARALLAX_IS_STATIC)
+	if (prefs.toggles_secondary & PARALLAX_IS_STATIC)
 		src << "Space will no longer move."
 	else
 		src << "Space will now move."
+
+/client/verb/toggle_safety_check()
+
+	set name = "Toggle Gun Safety Check"
+	set category = "Preferences"
+	set desc = "Toggles firing guns on intents other than help."
+
+	prefs.toggles_secondary ^= SAFETY_CHECK //Held in Parallax because we don't want to deal with an SQL migration right now.
+	prefs.save_preferences()
+	src << "You will [(prefs.toggles_secondary & SAFETY_CHECK) ? "no longer" : "now"] fire your weapon on intents other than help."
```

Comment:
Shouldnt this be hurt instead of help ?

Reply:
<< other than >> help is grammatically correct.